src/bindings/python/src/pyopenvino/core/common.hpp

```diff
@@ -47,6 +47,8 @@ const std::map<ov::element::Type, py::dtype>& ov_type_to_dtype();
 
 py::dtype get_dtype(const ov::element::Type& ov_type);
 
+std::map<int, ov::element::Type> init_num_to_ov_type();
```

Comment:
Is it required in header?
Is used internally to create `dtype_to_ov_type_mapping`, could only in .cpp in `anonymous` namespace.

Reply:
Discussed, this "TODO" will be a part of overall common file refactor.